docs/examples/README.md

```diff
@@ -0,0 +1,60 @@
+# Get started with examples
+
+Learn how to install Playwright, set up your dev environment to author Playwright scripts, and example recipes to bootstrap your scripts.
+
+## Installing Playwright
+
+Playwright is a Node.js library and can be acquired through the npm registry. Use npm or yarn to install Playwright in your Node.js project.
+
+```
+npm i playwright
+```
+
+Once installed, you can `require` Playwright in your Node.js scripts. 
+
+```js
+const playwright = require('playwright');
+
+(async () => {
+  const browser = await playwright.chromium.launch();
+  await browser.close();
+})();
+```
+
+## Setup dev environment
+
+Playwright scripts can be developed just like any other Node.js script. You can use the [Node.js debugger](https://nodejs.org/api/debugger.html) or [VS Code debugging](https://code.visualstudio.com/docs/nodejs/nodejs-debugging) to set breakpoints and get fine grained control over execution.
+
+### Running browsers for debugging
+
+By default, Playwright runs the browsers in headless mode. To see the browser UI, pass the `headless: false` flag while launching the browser. You can also use `slowMo` to slow down execution.
+
+```js
+  playwright.firefox.launch({ headless: false, slowMo: 50 });
+```
+
+It is also possible to open **browser dev tools** during execution, to inspect the DOM tree or network activity.
+
+<p align="center"><a href="https://user-images.githubusercontent.com/284612/77158031-c18e7e80-6a5f-11ea-8a71-4a4bea54f879.png"><img src="https://user-images.githubusercontent.com/284612/77158031-c18e7e80-6a5f-11ea-8a71-4a4bea54f879.png" width="500" alt="WebKit dev tools"></a></p>
+
+## Example recipes
+
+### [Authentication](authentication.js)
+
+This scripts logs in on GitHub.com through Chromium, and then reuses the login cookies state in WebKit. This recipe can be used to speed up tests by logging in once and reusing login state.
+
+<!--
+## Core concepts
+
+* Browser
+* Browser contexts
+* Pages and frames
+* Evaluate in page context
+-->
+
+<!--
+Other examples
+* Page navigation and wait for load
```

Comment:
* Filling a form
* Uploading a file
* Handling a popup, e.g. clicking "I accept" to close it